sdk/identity/identity/test/manual/src/index.tsx

```diff
@@ -5,7 +5,7 @@ import * as React from "react";
 import * as ReactDOM from "react-dom";
 
 import { InteractiveBrowserCredential, BrowserLoginStyle } from "@azure/identity";
-import { KeysClient, Key } from "@azure/keyvault-keys";
+import { KeyClient, Key } from "@azure/keyvault-keys";
```

Comment:
Module '"@azure/keyvault-keys/types/keyvault-keys"' doesn't have 'Key' defined, better remove it.

Reply:
We'll fully fix the tsx example later. The READMEs are needed for the upcoming release, but the samples can be updated after the release.